dask/dataframe/io/parquet.py

```diff
@@ -358,15 +358,21 @@ def _read_parquet_row_group(fs, fn, index, columns, rg, series, categories,
 def _write_partition_fastparquet(df, fs, path, filename, fmd, compression,
                                  partition_on):
     from fastparquet.writer import partition_on_columns, make_part_file
+    import fastparquet
+    from distutils.version import LooseVersion
     # Fastparquet mutates this in a non-threadsafe manner. For now we just copy
     # it before forwarding to fastparquet.
     fmd = copy.copy(fmd)
     if not len(df):
         # Write nothing for empty partitions
         rgs = None
     elif partition_on:
-        rgs = partition_on_columns(df, partition_on, path, filename, fmd,
-                                   fs.sep, compression, fs.open, fs.mkdirs)
+        if LooseVersion(fastparquet.__version__) >= LooseVersion('0.1.4'):
```

Comment:
No need for `LooseVersion` on the rhs, `LooseVersion` does this automatically on comparison.